autobotAI_integrations/integrations/google_chat/webhook_client.py

```
@@ -17,36 +17,51 @@ def send_raw(self, message):
         resp = request.urlopen(req)
 
     def send(self, title, body, buttons=[]):
```

Ruff (0.15.13)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)